pctasks/dataset/tests/test_dataset.py

```diff
@@ -31,54 +31,58 @@ def test_process_items() -> None:
     ds_config = template_dataset_file(DATASET_PATH)
     collection_config = ds_config.collections[0]
 
-    workflow = create_process_items_workflow(
-        ds_config,
-        collection_config,
-        chunkset_id="test-chunkset",
-        ingest=False,
-        target="test",
-    )
-    assert workflow.target_environment == "test"
+    with TemporaryDirectory() as tmp_dir:
```

Comment:
This is a large diff, but that's mostly from indentation. The real changes are

1. I moved these tasks context managers up
2. I iterate over use_existing_chunks=False/True